act/__init__.py

```diff
@@ -1,24 +1 @@
-import uuid
```

Comment:
This file is moved to act/api/__init__py but somehow it appears to be deletec (and created below) instead.

In addition the __init__.py directly under act adhers to the pkgutil based python packaging namespace.